source/reference/json.txt

```diff
@@ -1,8 +1,106 @@
+.. _json-cluster-config:
 
 ======
 |json|
 ======
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The {+atlas-cli+} accepts ``.json`` configuration files. You can use a 
+configuration file to specify settings when using the {+atlas-cli+} to:
+
+- :ref:`Create <atlas-clusters-create>`, :ref:`update <atlas-clusters-update>`, 
+  or :ref:`upgrade a cluster <atlas-clusters-upgrade>`.
+- :ref:`Update a cloud backup schedule <atlas-backups-schedule-update>`.
+- :ref:`Create <atlas-clusters-search-indexes-create>` or 
+  :ref:`update an Atlas Search Index <atlas-clusters-search-indexes-update>`.
+
+Usage
+-----
+
+To learn which settings are accepted by the {+atlas-cli+}, view the request 
+body schema and request samples in the :ref:`Atlas Admin API Specification 
+<api-resources-spec>` for the endpoint that corresponds to the {+atlas-cli+} 
+command:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 50 50
+          
+  * - {+atlas-cli+} Command
+    - API Endpoint
+
+  * - :ref:`atlas-clusters-create`
+    - :oas-atlas-op:`Create One Cluster </createLegacyCluster>`
+
+  * - :ref:`atlas-clusters-update`
+    - :oas-atlas-op:`Update Configuration of One Cluster </updateClusterConfiguration>`
+
+  * - :ref:`atlas-clusters-upgrade`
+    - :oas-atlas-op:`Upgrade One Shared-tier Cluster </upgradeSharedCluster>`
+
+  * - :ref:`atlas-backups-schedule-update`
+    - :oas-atlas-op:`Update Cloud Backup Schedule for One Cluster </updateBackupSchedule>`
+
+  * - :ref:`atlas-clusters-search-indexes-create`
+    - :oas-atlas-op:`Create One Atlas Search Index </createAtlasSearchIndex>`
+
+  * - :ref:`atlas-clusters-search-indexes-update`
+    - :oas-atlas-op:`Update One Atlas Search Index </updateAtlasSearchIndex>`
+
+When you run the command in your terminal, specify the ``--file`` option 
+and provide the path to the |json| configuration file that defines your 
+desired settings. 
+
+Example
+~~~~~~~
+
+For example, consider the following configuration file
+to enable backups and add a label:
+
+.. code-block:: sh
+   :caption: /example.json
+
+   {
+     "backupEnabled" : true,
+     "labels": [
+       {
+         "key": "foo",
+         "value": "bar"
```

Comment:
Per the style guide/checked with Corry, in code examples we should use a placeholder here https://www.mongodb.com/docs/meta/style-guide/style/code-examples/
```suggestion
         "key": "<myKey>",
         "value": "<myValue>"
```